src/main/java/io/jenkins/plugins/opentelemetry/job/log/OtelLogStorageFactory.java

```diff
@@ -86,35 +86,23 @@ private LogStorage forExec(@NonNull Queue.Executable exec) {
         if (exec instanceof Run<?, ?> run && run.getAction(MonitoringAction.class) != null) {
             // it's a pipeline with monitoring data
             logger.log(Level.FINEST, () -> "forExec(" + run + ")");
-            ret = new OtelLogStorage(run, getOtelTraceService(), tracer);
+            ret = new OtelLogStorage(run, otelTraceService, tracer);
         }
         return ret;
     }
 
-    /**
-     * Workaround dependency injection problem. @Inject doesn't work here
-     */
-    @NonNull
-    private JenkinsControllerOpenTelemetry getJenkinsControllerOpenTelemetry() {
-        if (jenkinsControllerOpenTelemetry == null) {
-            jenkinsControllerOpenTelemetry = JenkinsControllerOpenTelemetry.get();
+    @Extension()
+    @Symbol("opentelemetry")
```

Comment:
I think `camelCase` here:

```suggestion
    @Symbol("openTelemetry")
```